src/cli/readme.MD

```diff
@@ -0,0 +1,80 @@
+# Cloud Foundry CLI AutoScaler Plugin
+
+AutoScaler plugin provides the command line interface to manage AutoScaler service policies, retrieve metrics and scaling histories.
+
+## Command List
+
+| Command | Description |
+|---------|-------------|
+| [as-api](#cf-as-api) | Set or view AutoScaler service API endpoint |
+| [as-policy](#cf-as-policy) | Retrieve the scaling policy of an application |
+| [attach-as-policy](#cf-attach-as-policy) | Attach a scaling policy to an application |
+| [detach-as-policy](#cf-detach-as-policy) | Detach the scaling policy from an application |
+| [as-metrics](#cf-as-metrics) | Retrieve the metrics of an application |
+| [as-histories](#cf-as-histories) | Retrieve the scaling histories of an application|
+
+## Command usage
+
+### `cf as-api`
+
+Set or view AutoScaler service API endpoint
+
+```
+cf as-api [URL] [--unset] 
+```
+
+**OPTIONS**
+- `--unset`: Unset the api endpoint
+
+### `cf as-policy` 
+
+Retrieve the scaling policy of an application
+
+```
+cf as-policy APP_NAME [-f PATH_TO_FILE]
+```
+
+**OPTIONS**
+- `-f` : dump the policy to a file in JSON format
+
+### `cf attach-as-policy` 
+
```

Comment:
Just a suggestion, good to have "aas" (app-autoscaler) instead of "as". When  using it in "attach-as-policy", it sounds like having policy as an option. Also is it good to have "aas" prepended in all commands as in
cf aas-api
cf aas-attach-policy
cf aas-detach-policy
and so on

Reply:
I think "as" is concise, but it is a good idea to have "as" as the prefix.  how about 
cf as-policy-show
cf as-policy-attach
cf as-policy-detach

Reply:
is it odd to abbreviate here though when cli commands can also have abbreviations? i.e. maybe it should be `cf attach-autoscaler-policy` and then we can provide a short-form like `cf aap` too. (which fits with things like `cf create-service-instance` + `cf csi` rather than `cf create-s-instance` iyswim)

Reply:
(either way I think we need to avoid `as` definitely because there's a lot of confusion there -- I read `detach-as-policy` as "detach as policy" as in the English word "as")

Reply:
Yes thats the point I was trying to make about "as".

Reply:
agreed.  'as' may cause confusion.  Let us use full name autoscaler, and also provide short name for each command